fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt

```diff
@@ -138,6 +140,14 @@ class SecretSettingsFragment : PreferenceFragmentCompat() {
         }
     }
 
+    private fun setupTabStripPreference() {
```

Comment:
this doesn't necessarily need to be in its own one-off function

Reply:
Shaped by detekt, so we don't add detekt suppression here. Think every item here can be a one off function to prevent it further but that's outside the scope of this PR.